### DISCUSSION TITLE
Run Javascript code when clicking the table cell item

### DISCUSSION THREAD

**Discussion by GollaSaiVenkatesh** [grafana/grafana#57649] [Dashboards, Panels & Visualizations]
It would be awesome if table panel allows javascript to be written inside cell
One usecase is like adding another column for deleting a row  which contains a delete button and when user clicks it , it deletes the row and alert that it is successful and reloads the page which can be easily done using javascript

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
